java/README.md

```diff
@@ -31,20 +31,28 @@ PLAID_PRODUCTS=[PRODUCTS] \
 PLAID_COUNTRY_CODES='US' \
 java -jar target/quickstart-1.0-SNAPSHOT.jar server config.yml
 
-# Go to http://localhost:8080
+# Go to http://localhost:8000
 ```
 
+### Link token creation and server-side configuration.
+The recommended way to initialize Plaid Link is to pass Plaid Link initialization
```

Comment:
Let's also point to the new docs `https://plaid.com/docs/#create-link-token` (The fragment part doesn't work yet but will once we merge the docs updates)